### DISCUSSION TITLE
Global scheduling with one local exception

### DISCUSSION THREAD

**Discussion by Omikorin** [renovatebot/renovate#26417] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

Hi!

I want to schedule each dependency on `earlyMondays` but not for one distinct dependency: a logic layer underneath our library. I am happy to reduce update noise, but one particular monorepo needs to be updated whenever possible (for continuous integration).

I have this config, and after reading the documentation, I still don't know how to overwrite a global schedule for one package in the default "always on/at instant" config at "????" place below.

Is it possible to overwrite a global schedule at all?

Repo for reference [here](https://github.com/chakra-ui/ark).

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": ["config:base", ":semanticCommitTypeAll(chore)", "schedule:earlyMondays"],
  "labels": ["dependencies"],
  "packageRules": [
    {
      "matchUpdateTypes": ["minor", "patch", "pin", "digest"],
      "automerge": true
    },
    {
      "extends": "monorepo:zag-js",
      "schedule": "????"
    },
    {
      "groupName": "lucide-icons",
      "matchPackagePrefixes": ["lucide"]
    }
  ],
  "ignoreDeps": ["@types/node"]
}
```

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Try "at any time"

**Reply by Omikorin** [reply]
Thanks, it works. Would be great to add this to the docs

**Reply by rarkins** [reply]
PRs welcome
